Configuration/Services.yaml

```diff
@@ -4,17 +4,37 @@ services:
     autoconfigure: true
     public: false
 
+  # Parameters
+  parameters:
+    telephonedirectory.email_service.settings:
+      pidOfDetailPage: 123
```

Comment:
Why do we define the details page setting twice? In TypoScript and in Services.yaml? Maybe OK for telephonedirectory, but should not default for our other extensions. Maybe migrate to Site Settings? Or read TS config in MailService or transfer PID as argument?